### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by paulfantom** [kubernetes-monitoring/kubernetes-mixin#385]
When running `promtool test rules tests.yaml` with promtool in version 2.16.0 I get following:
```
  FAILED:
    alertname:KubePersistentVolumeUsageCritical, time:4m0s, 
        exp:"[Labels:{alertname=\"KubePersistentVolumeUsageCritical\", job=\"kubelet\", namespace=\"monitoring\", persistentvolumeclaim=\"somepvc\", severity=\"critical\"} Annotations:{message=\"The PersistentVolume claimed by somepvc in Namespace monitoring is only 1.562% free.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubepersistentvolumeusagecritical\"}]", 
        got:"[]"
    alertname:KubePersistentVolumeFullInFourDays, time:1h1m0s, 
        exp:"[Labels:{alertname=\"KubePersistentVolumeFullInFourDays\", job=\"kubelet\", namespace=\"monitoring\", persistentvolumeclaim=\"somepvc\", severity=\"critical\"} Annotations:{message=\"Based on recent sampling, the PersistentVolume claimed by somepvc in Namespace monitoring is expected to fill up within four days. Currently 1.263% is available.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubepersistentvolumefullinfourdays\"}]", 
        got:"[]"
    alertname:KubeletTooManyPods, time:15m0s, 
        exp:"[Labels:{alertname=\"KubeletTooManyPods\", node=\"minikube\", severity=\"warning\"} Annotations:{message=\"Kubelet 'minikube' is running at 98.18% of its Pod capacity.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubelettoomanypods\"}]", 
        got:"[]"
    alertname:KubeletPlegDurationHigh, time:10m0s, 
        exp:"[Labels:{alertname=\"KubeletPlegDurationHigh\", instance=\"10.0.2.15:10250\", node=\"minikube\", quantile=\"0.99\", severity=\"warning\"} Annotations:{message=\"The Kubelet Pod Lifecycle Event Generator has a 99th percentile duration of 10 seconds on node minikube.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubeletplegdurationhigh\"}]", 
        got:"[]"
    alertname:KubeNodeReadinessFlapping, time:18m0s, 
        exp:"[Labels:{alertname=\"KubeNodeReadinessFlapping\", node=\"minikube\", severity=\"warning\"} Annotations:{message=\"The readiness status of node minikube has changed 10 times in the last 15 minutes.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubenodereadinessflapping\"}]", 
        got:"[]"
```

Can we update promtool to latest version to be used in CI (cc: @csmarchbanks)?

**Comment by lilic** [comment]
I have been seeing something similar as well in this PR https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/368 when trying to adjust the alert 🤔

**Comment by csmarchbanks** [comment]
Created #388 to use the same builder as in the prometheus-operator rather than hosting our own. Any concerns with doing that?

**Comment by metalmatze** [comment]
Closing as I merged #388
